### PR TITLE
Fix messages

### PR DESCRIPTION
**Pull request by LeSpocky**
Minor things found with clang and codespell.

### CODE REVIEW

File: Stack/transport/https/opcua_httpsstream.c [hunk 1]
```diff
@@ -2711,7 +2711,7 @@ OpcUa_InitializeStatus(OpcUa_Module_HttpStream, "DataReady");
 
                 pHttpInputStream->State = OpcUa_HttpsStream_State_Headers;
             }
-            /* fall thru */
+            /* fall through */
```
[the remark this answers is not in the document]
Comment (LeSpocky) [reply Stack/transport/https/opcua_httpsstream.c]: GCC uses regular expressions to match those keywords. I verified the changed comment would match the regex described in the [GCC documentation](https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html) and successfully tested with GCC 7 (Debian sid/unstable).

[the remark this answers is not in the document]
Comment (bernd-edlinger) [reply Stack/transport/https/opcua_httpsstream.c]: okay.

[the remark this answers is not in the document]
Comment (bernd-edlinger) [reply Stack/transport/https/opcua_httpsstream.c]: Please revert those...

[the remark this answers is not in the document]
Comment (LeSpocky) [reply Stack/transport/https/opcua_httpsstream.c]: Done. I force pushed the updated version.

File: Stack/stackcore/opcua_connection.h [hunk 2]
```diff
@@ -280,13 +280,13 @@ struct _OpcUa_Connection
     /*! @brief Aborts writing an outgoing message. */
     OpcUa_Connection_PfnAbortSendRequest* AbortSendRequest;
 
-    /*! @brief Retrive the recieve buffer size of a particular connection. */
+    /*! @brief Retrive the receive buffer size of a particular connection. */
```
Comment (bernd-edlinger) [inline Stack/stackcore/opcua_connection.h]: That is funny, isn't that "Retrieve"

File: Stack/securechannel/opcua_tcpsecurechannel.h [hunk 3]
```diff
@@ -41,7 +41,7 @@ OpcUa_StatusCode OpcUa_TcpSecureChannel_Create(OpcUa_SecureChannel**   ppSecureC
 /**
   @brief Deletes a securechannel object.
 
-  A secure connection is always layed on top of a regular connection.
+  A secure connection is always laid on top of a regular connection.
```
Comment (bernd-edlinger) [inline Stack/securechannel/opcua_tcpsecurechannel.h]: would "layered" be a better word here?

File: Stack/proxystub/clientproxy/opcua_clientapi.c [hunk 4]
```diff
@@ -124,7 +124,7 @@ OPCUA_EXPORT OpcUa_StatusCode OpcUa_ClientApi_FindServers(
         *a_pServers        = pResponse->Servers;
     }
 
-    /* memory contained in the reponse objects is owned by the caller */
+    /* memory contained in the response objects is owned by the caller */
```
Comment (bernd-edlinger) [inline Stack/proxystub/clientproxy/opcua_clientapi.c]: Oh, this is generated code, please keep this change,
but we need to ask @randy-armstrong to change the generator script.

File: Stack/stackcore/opcua_connection.h (hunk 2 above)
Comment (LeSpocky) [reply Stack/stackcore/opcua_connection.h]: Yes, I think so. As stated above, this fixes only what [codespell](https://github.com/codespell-project/codespell) found, and I used v1.8 from Debian 9 (stretch). The recent upstream version is v1.13, I did not try that.

File: Stack/securechannel/opcua_tcpsecurechannel.h (hunk 3 above)
Comment (LeSpocky) [reply Stack/securechannel/opcua_tcpsecurechannel.h]: Yes, I think so.

Comment (bernd-edlinger) [reply Stack/securechannel/opcua_tcpsecurechannel.h]: Okay, then, please change that as well.

File: Stack/stackcore/opcua_connection.h (hunk 2 above)
Comment (bernd-edlinger) [reply Stack/stackcore/opcua_connection.h]: No problem, but please change that as well.

File: Stack/proxystub/clientproxy/opcua_clientapi.c (hunk 4 above)
Comment (LeSpocky) [reply Stack/proxystub/clientproxy/opcua_clientapi.c]: Okay, the last line in the file indicates that it is autogenerated:
```
/* This is the last line of an autogenerated file. */
```
But speaking for myself: hiding the fact, that the project contains autogenerated files in a comment in the last line of some >5000 lines of code file with no hint what it was generated from, is in no way obvious!

**Comment by LeSpocky** [comment]
For the updated second commit I used codespell 1.13.0 now, which covers "retrieve" and some more spelling mistakes, which were not catched before. I also reverted all changes to files containing the keyword 'autogenerated'.

**Pull request merged**